lfs.c

```diff
@@ -4467,6 +4476,7 @@ static int lfs1_mount(lfs_t *lfs, struct lfs1 *lfs1,
         lfs->lfs1->root[1] = LFS_BLOCK_NULL;
 
         // setup free lookahead
+        // TODO should this also call lfs_setup_invalid_lookahead_buffer(lfs); the free.off is different in the current version of lfs
```

Comment:
Yeah all three entry points are a bit uniquely special, that's because we need to trick the allocator into always giving the superblock blocks 0 and 1 (off = 0).

You could replace this with `lfs_setup_invalid_lookahead_buffer` if you init lfs->seed with 0, but that may be too clever. It's likely not worth changing since modifying v1 code carries has more risks than rewards.

Reply:
Thanks for the feedback, I'll leave it as is an remove the TODO